RecoTracker/FinalTrackSelectors/plugins/TrackCutClassifier.cc

```diff
@@ -36,13 +38,36 @@ namespace {
     
   inline float chi2n(reco::Track const & tk) { return tk.normalizedChi2();}
 
+  inline float relPtErr(reco::Track const & tk) {
+    return  float(tk.ptError())/float(tk.pt());
```

Comment:
Is protection against `tk.pt() == 0` needed here in the case of bad or corrupted tracks?